src/app/components/EmailInput/EmailInput.tsx

```diff
@@ -1,11 +1,40 @@
 import { FC } from "react";
 import { EmailInputProps } from "./EmailInput.types";
+import styles from "./EmailInput.module.css";
+
+export const EmailInput: FC<EmailInputProps> = ({ ...props }) => {
+
+  const { testId, required, label, value, error, helperText, placeholder, disabled, onChange } = props;
 
-export const EmailInput: FC<EmailInputProps> = ({testId, ...props}: EmailInputProps) => {
   return (
-    <>
-      <input type="email" {...props}  data-testid={testId}/>
-      {/* TODO: Implement EmailInput label, helpertext and error */}
-    </>
+    <div
+      className={styles.emailInput}
+      data-testid={testId}
+    >
+      <label
+        className={`${!!error ? styles.emailInputLabelError : ""} ${styles.emailInputLabel}`}
+        htmlFor={`${testId}`}>
```

Comment:
`testId` should be used only for testing purposes, and not in production code. Instead, use the `id` property (extend interface if needed)

Reply:
Add id prop and extended interface.